metacat-functional-tests/src/functionalTest/groovy/com/netflix/metacat/MetacatFunctionalSpec.groovy

```diff
@@ -1183,6 +1177,176 @@ class MetacatFunctionalSpec extends Specification {
         name << TestCatalogs.getAllDatabases(TestCatalogs.getCanCreateTable(TestCatalogs.ALL))
     }
 
+    def 'test get WAP pattern partitions'() {
```

Comment:
isn't WAP an internal only construct?

Reply:
I will change the WAP as AUDIT. This is nflx specific, we can make it configurable.